Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/launcher/LauncherActivityViewModel.kt

```diff
@@ -83,9 +83,11 @@ class LauncherActivityViewModel @AssistedInject constructor(
 
     private fun checkForRoot() = launch {
         Timber.tag(TAG).d("checkForRoot()")
-        when (rootDetectionCheck.isRooted()) {
-            true -> events.postValue(LauncherEvent.ShowRootedDialog)
-            false -> checkForUpdate()
+        with(rootDetectionCheck) {
+            when (shouldShowRootInfo && isRooted()) {
```

Comment:
Minor: IMO it is nicer to have the `isRooted()` check within the `shouldShowRootInfo` property. Then the `ViewModel` only needs to ask the `RootDetectionChecker` if it should show the info and doesn't need to additionally check the root status itself.

However we can also leave it as is due to time pressure.